docs/api/model.md

```diff
@@ -22,10 +22,10 @@ Calls the MongoDB [`aggregate()`](https://mongodb.github.io/node-mongodb-native/
 
 **Parameters:**
 
-| Name       | Type                             | Attribute |
```

Comment:
Please rebase on latest `main` commit and `yarn` to generate these docs properly.